test/user.test.js

```diff
@@ -214,6 +215,95 @@ describe('User', function() {
       assert(u2.password === u1.password);
     });
 
+    it('invalidates the user\'s accessToken when the user is deleted By id', function(done) {
+      var usersId;
+      async.series([
+        function(next) {
+          User.create({ email: 'b@c.com', password: 'bar' }, function(err, user) {
+            usersId = user.id;
+            if (err) return done (err);
+            next();
+          });
+        },
+        function(next) {
+          User.login({ email: 'b@c.com', password: 'bar' }, function(err, accessToken) {
+            if (err) return done (err);
+            assert(accessToken.userId);
+            console.log('acessToken.userId: ', accessToken.userId);
+            console.log('>>>user: ', usersId);
+            next();
+          });
+        },
+        function(next) {
+          User.deleteById(usersId, function(err) {
+            if (err) return done (err);
+            next();
+          });
+        },
+        function(next) {
+          User.findById(usersId, function(err, userFound)  {
+            if (err) return done (err);
+            expect(userFound).to.equal(null);
+            AccessToken.find({ where: { userId: usersId }}, function(err, tokens) {
+              if (err) return done(err);
+              expect(tokens.length).to.equal(0);
+              console.log('tokens: ', tokens);
+              next();
+            });
+          });
+        },
+      ], function(err) {
+        if (err) return done (err);
+        console.log('series functions are done');
+        done();
+      });
+    });
+
+    it('invalidates the user\'s accessToken when the user is deleted all', function(done) {
+      var usersId, accessTokenId;
+      async.series([
+        function(next) {
+          User.create({ email: 'b@c.com', password: 'bar' }, function(err, user) {
+            usersId = user.id;
+            if (err) return done (err);
+            next();
+          });
+        },
+        function(next) {
+          User.login({ email: 'b@c.com', password: 'bar' }, function(err, accessToken) {
+            accessTokenId = accessToken.userId;
+            if (err) return done (err);
+            assert(accessTokenId);
+            console.log('acessToken.userId: ', accessTokenId);
+            console.log('>>>user: ', usersId);
+            next();
+          });
+        },
+        function(next) {
+          User.deleteAll({ id: accessTokenId }, function(err) {
```

Comment:
This looks suspicous to me, you are deleting all users that have id same as access token id.

I think you should create two users with a property that has same value (e.g. `name` or similar) and then delete them via `User.deleteAll({ where: { name: '...' } })`